ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -31,6 +31,25 @@ See http://www.irchelp.org/clients/ for more recommendations.
 
 Once you’ve joined IRC, pick any channel (#rebuild, #henlo, etc) and say hi!
 
+## Get familiar with the command line
+All of [[our servers|doc staff/backend/servers]] run Linux, and we interact with
+them over the command line. There are a lot of online resources about using the
+Linux command line, so if you're confused about something, try Googling it!
+
+Here are a few commands you will want to know about:
+- `cd` (change directory): move from one directory (folder) to another
+- `ls` (list): list the files in the current directory
+- `cat` (concatenate): print file contents
+- `less`: read long files
+- `mv` (move): move or rename files
+- `cp` (copy): copy files
+- `rm` (remove): delete files or directories
+
+Go to our [[command reference|doc services/shell/commands]] page and try running some commands. If
```

Comment:
I think it's fine to specify here that we want people to log into the "tsunami" server, and link to the [`services/shell`](https://www.ocf.berkeley.edu/docs/services/shell/) docs.

Reply:
The command reference page is really nice and we should use it more for teaching, which is why I linked it. I trust that people who have used SSH before will know that they can connect to tsunami directly.

Perhaps it would be better to change `services/shell` to show link more prominently to `services/shell/commands`, and then your suggestion would work better.